### ISSUE TITLE
secrets "trow-tls" not found

### ISSUE THREAD

**Issue by etoiledemer** [ContainerSolutions/trow#292]
Can someone explain what I need to do with this:

```
Tue Jan 18 16:33:34 UTC 2022

Generating new certificate

Getting IP of trow service

POD NAMESPACE: kube-public

POD NAME: trow-deploy-744888894b-b8cgj

POD IP: 10.1.0.6

SERVICE IP: 10.104.127.176

Error from server (NotFound): secrets "trow-tls" not found

2022/01/18 16:33:34 [INFO] generate received request

2022/01/18 16:33:34 [INFO] received CSR

2022/01/18 16:33:34 [INFO] generating key: rsa-4096

2022/01/18 16:33:39 [INFO] encoded CSR

Error from server (NotFound): certificatesigningrequests.certificates.k8s.io "trow.kube-public" not found

error: unable to recognize "STDIN": no matches for kind "CertificateSigningRequest" in version "certificates.k8s.io/v1beta1"

```

**Comment by etoiledemer** [comment]
**To Reproduce**


clean install of Docker Mac and Kubernetes...
Docker Desktop
```
git clone..
cd ./trow/quick-install
./install.sh
```




**Trow Info**
 - Install method (quick-install, Helm, Kustomize)
 - Version/Image Name (e.g. `containersol/trow:0.3`, can be obtained from `kubectl describe`) 

```
docker/desktop-kubernetes            kubernetes-v1.22.4-cni-v0.8.5-critools-v1.17.0-debian   493a106d3678   2 months ago    294MB
k8s.gcr.io/kube-apiserver            v1.22.4                                                 8a5cc299272d   2 months ago    128MB
k8s.gcr.io/kube-scheduler            v1.22.4                                                 721ba97f54a6   2 months ago    52.7MB
k8s.gcr.io/kube-controller-manager   v1.22.4                                                 0ce02f92d3e4   2 months ago    122MB
k8s.gcr.io/kube-proxy                v1.22.4                                                 edeff87e4802   2 months ago    104MB
k8s.gcr.io/etcd                      3.5.0-0                                                 004811815584   7 months ago    295MB
k8s.gcr.io/coredns/coredns           v1.8.4                                                  8d147537fb7d   7 months ago    47.6MB
docker/desktop-vpnkit-controller     v2.0                                                    8c2c38aa676e   8 months ago    21MB
docker/desktop-storage-provisioner   v2.0                                                    99f89471f470   8 months ago    41.9MB
k8s.gcr.io/pause                     3.5                                                     ed210e3e4a5b   10 months ago   683kB
containersol/trow                    init                                                    2c5e2a7afa9b   16 months ago   1.23GB
```
```
git clone..
cd ./trow/quick-install
./install.sh
```
**Kubernetes**
 - Kubernetes distro/host (EKS, GKE, Kind, Minikube etc)
 - Kubernetes version (e.g. 1.19)
 - Container Runtime (e.g. Docker, can be found via `kubectl get nodes -o wide`)
```
macOS
Docker Desktop
1.22.4
```

**Comment by etoiledemer** [comment]
@amouat any idea?

**Comment by etoiledemer** [comment]
OK fixed by downgrading to Docker Desktop 4.2.0 with Kubernetes 1.21

Trow maybe is not compatible with Kubernetes 1.22.4

**Comment by amouat** [comment]
It looks like you're using the quick install. I suspect new versions of Docker Desktop use the k8s containerd driver :(
